crypto.go

```diff
@@ -226,16 +229,16 @@ func (o *OnionErrorDecrypter) DecryptError(encryptedData []byte) (*btcec.PublicK
 		// If the MAC matches up, then we've found the sender of the
 		// error and have also obtained the fully decrypted message.
 		realMac := h.Sum(nil)
-		if hmac.Equal(realMac, expectedMac) && sender == nil {
-			sender = o.circuit.PaymentPath[i]
+		if hmac.Equal(realMac, expectedMac) && sender == 0 {
+			sender = i + 1
```

Comment:
Can simply be `+=`

Reply:
I don't understand this suggestion. What would you change this line into exactly?

Reply:
Ah nvm, misread the line the first time around.